plexus-compilers/plexus-compiler-javac/src/main/java/org/codehaus/plexus/compiler/javac/JavacCompiler.java

```diff
@@ -635,6 +635,11 @@ protected CompilerResult compileOutOfProcess( CompilerConfiguration config, Stri
         {
             returnCode = CommandLineUtils.executeCommandLine( cli, out, out );
 
+            if ( log.isDebugEnabled() )
+            {
+                log.debug( "Compiler output:\n{}", out.getOutput() );
```

Comment:
Should this perhaps use the protected [`AbstractCompiler#EOL`](https://github.com/codehaus-plexus/plexus-compiler/blob/97542e84a2aba410656d5a29ac4dbe79cc889d86/plexus-compiler-api/src/main/java/org/codehaus/plexus/compiler/AbstractCompiler.java#L46) constant, rather than `\n`? And if so, likewise below?

Reply:
good call